### ISSUE TITLE
Instance not starting offline 

### ISSUE THREAD

**Issue by tuckpendleton** [MultiMC/MultiMC5#866]
I don't have access to Internet only via phone tethering it start when online but in offline I get this
```
MultiMC version: 0.4.5
Minecraft folder is:
C:/Users/Joseph Abramov/Desktop/MultiMC/instances/1.81/minecraft
Java path is:
C:/Program Files/Java/jdk1.7.0_75/bin/javaw.exe
Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms512m, -Xmx4096m, -XX:PermSize=256m, -Duser.language=en, -jar, C:/Users/Joseph Abramov/Desktop/MultiMC/jars/NewLaunch.jar]
Using onesix launcher.
Not enough arguments.
org.multimc.NotFoundException
at org.multimc.ParamBucket.all(ParamBucket.java:43)
at org.multimc.onesix.OneSixLauncher.processParams(OneSixLauncher.java:55)
at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:302)
at org.multimc.EntryPoint.listen(EntryPoint.java:165)
at org.multimc.EntryPoint.main(EntryPoint.java:54)
Exiting with -1
Minecraft exited with exitcode -1.
```
Any help ??? Thanks

**Comment by tuckpendleton** [comment]
That code is from another error from another site only difference from my error is I'm using 1.7.10 If that helps

**Comment by peterix** [comment]
Code:
https://github.com/MultiMC/MultiMC5/blob/stable/depends/launcher/org/multimc/onesix/OneSixLauncher.java#L55

It's missing the list of native libraries to extract.

Can you post the MultiMC log? Just trigger the problem and without closing/restarting MultiMC, post the MultiMC.0.log. Make sure to remove any personal info from it (it can have your email address).

**Comment by tuckpendleton** [comment]
Ok will do as soon as back from work thanks.

**Comment by tuckpendleton** [comment]
Just seems strange I can start online mode exit unplug usb from phone try to start offline and that error comes up.

**Comment by peterix** [comment]
Well, offline mode is one of those things that get broken easily.

You could try switching to the 'develop' version too - it has a lot of extra fixes on top of 'stable'.

**Comment by tuckpendleton** [comment]
Ok will try the develop version, is that on the main site to download? Thanks

**Comment by peterix** [comment]
You can switch the update channels in the settings (and update after that, obviously).

**Comment by Loetkolben** [comment]
Just go into MMC's settings, and change the update channel to "Development", than restart MMC and you should be asked to update.

**Comment by tuckpendleton** [comment]
http://paste.ee/p/HtRv6. this is the only log that shows up, the "other logs" is empty, the game runs fine with my phone plugged in tethering but as soon as i disconnect and retry to start this pops up?

**Comment by tuckpendleton** [comment]
its not the end of the world if i have to keep plugging phone in to start the game just, that for a long time now the offline mode has always worked, now out of the blue it stop. thanks

**Comment by p-schneider** [comment]
I had the same error like @tuckpendleton and found a workaround. My log looks like his (mine begins with `MultiMC version: 0.4.5-dev-470`).

It looks like `OneSixInstance::reloadProfile()` is never called if i try to start a new game (e.g. vanilla 1.8.3) without Internet Connection.
If I click on Edit Instance the Version tab is empty. I have to click Reload to get the Versions (e.g. Minecraft 1.8.3 JWJGL 2.9.1). After that I am able to launch that instance. If I switch instances or restart MultiMC I have to click Edit Instance -> (Version) -> Reload again before I can successfully start the game.

Adding the line `m_selectedInstance->reload();` before `launchInstance(m_selectedInstance, session, profiler);` in [MainWindow.cpp Line 1707](https://github.com/MultiMC/MultiMC5/blob/28aa8f342ebaf34655ed0c1dd59851adb224fc64/application/MainWindow.cpp#L1707) fixed the launching problem for me, but I'm sure the reload should be called way earlier so the Version Tab in Edit Instance would also work properly.

Using the unstable version clicking Reload doesn't reveal the Versions and I couldn't get the game to launch.
Using the stable version woks fine (no workaround needed).

**Comment by tuckpendleton** [comment]
Ok thanks for giving me something to try appreciate it will try it asap thanks

**Comment by p-schneider** [comment]
As of `MultiMC version: 0.4.5-dev-472` Offline Mode is now working for me. I'm sure it will also work for @tuckpendleton so I think this Issue can now be closed. Thanks to @peterix for implementing the fix in 3507ccaf50e22b68850f5b8a2072a56e218691f4.

**Comment by tuckpendleton** [comment]
Just look at your advice and also under edit instance the version was empty, clicking reload and started up offline straight away many thanks for all the help
